Uchu.StandardScripts/AvantGardens/TargetFilter.cs

```diff
@@ -1,23 +1,42 @@
+using System.Linq;
+using System.Threading.Tasks;
 using Uchu.Core.Resources;
 using Uchu.World;
 using Uchu.World.Scripting.Native;
 
 namespace Uchu.StandardScripts.AvantGardens
 {
     /// <summary>
-    /// Native implementation of scripts/02_client/map/ag/l_ag_plunger_target.lua
+    /// Script to show/hide targets based on whether the player has the relevant mission
     /// </summary>
-    [ScriptName("l_ag_plunger_target.lua")]
-    public class TargetFilter : ObjectScript
+    [ZoneSpecific(1100)]
```

Comment:
What was the purpose of making this `ZoneSpecific` again?

Reply:
The ScriptName it used refers to a client script. After the changes on this branch, these don't get loaded anymore. No server script name is attached to these objects, so I've reverted this script to an older NativeScript implementation.